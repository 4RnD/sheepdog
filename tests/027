#!/bin/bash

# Test sheep recovery logic

seq=`basename $0`
echo "QA output created by $seq"

here=`pwd`
tmp=/tmp/$$
status=1        # failure is the default!

# get standard environment, filters and checks
. ./common.rc
. ./common.filter

_cleanup

for i in `seq 0 3`; do
    _start_sheep $i
done

_wait_for_sheep "4"

$COLLIE cluster format -c 2

$COLLIE vdi create test0 40M
$COLLIE vdi create test1 40M

_kill_sheep 3

_wait_for_sheep_recovery 0

find $STORE -name '80fd32fc00000000'
