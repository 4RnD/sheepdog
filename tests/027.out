QA output created by 027
using backend farm store
/tmp/sheepdog/0/obj/80fd32fc00000000
/tmp/sheepdog/3/obj/80fd32fc00000000
/tmp/sheepdog/1/obj/80fd32fc00000000
